MovingBlocks/Terasology
# Removed Title and Default fonts from engine

### Contains
Fixes one part of issue #2472 

### How to test
run the application

Brief description of how to test / confirm this PR before mergin
Removes title and default fonts from engine folder
